openslides/motions/serializers.py

```diff
@@ -138,36 +139,76 @@ class MotionSerializer(ModelSerializer):
     """
     Serializer for motion.models.Motion objects.
     """
-    versions = MotionVersionSerializer(many=True, read_only=True)
     active_version = PrimaryKeyRelatedField(read_only=True)
-    submitter = MotionSubmitterSerializer(many=True, read_only=True)
-    supporter = MotionSupporterSerializer(many=True, read_only=True)
+    log_messages = MotionLogSerializer(many=True, read_only=True)
+    polls = MotionPollSerializer(many=True, read_only=True)
+    reason = CharField(allow_blank=True, required=False, write_only=True)
     state = StateSerializer(read_only=True)
+    submitter = MotionSubmitterSerializer(many=True, required=False)
+    supporter = MotionSupporterSerializer(many=True, required=False)
+    text = CharField(write_only=True)
+    title = CharField(max_length=255, write_only=True)
+    versions = MotionVersionSerializer(many=True, read_only=True)
     workflow = SerializerMethodField()
-    polls = MotionPollSerializer(many=True, read_only=True)
-    log_messages = MotionLogSerializer(many=True, read_only=True)
 
     class Meta:
         model = Motion
         fields = (
             'id',
             'identifier',
-            'identifier_number',
-            'parent',
-            'category',
-            'tags',
+            'title',
+            'text',
+            'reason',
             'versions',
             'active_version',
+            'parent',
+            'category',
             'submitter',
             'supporter',
             'state',
             'workflow',
+            'tags',
             'attachments',
             'polls',
             'log_messages',)
+        read_only_fields = ('parent',)  # Some other fields are also read_only. See definitions above.
 
     def get_workflow(self, motion):
         """
         Returns the id of the workflow of the motion.
+
+        You can also include 'workflow' in create and update requests, see
+        the methods 'create' and 'update' below.
         """
         return motion.state.workflow.pk
+
+    def create(self, validated_data):
+        """
+        Customized method to create a new motion from some data.
+        """
+        motion = Motion()
+        motion.title = validated_data['title']
+        motion.text = validated_data['text']
+        motion.reason = validated_data.get('reason', '')
+        motion.identifier = validated_data.get('identifier')
+        motion.category = validated_data.get('category')
+        motion.workflow = validated_data.get('workflow', int(config['motion_workflow']))
+        motion.reset_state()
```

Comment:
Please have a look in the save method (which you call two lines later). If state is None, then reset_state() is called.